docs/widgetbook-cloud/reviews.mdx

```diff
@@ -4,22 +4,67 @@ Widgetbook Cloud introduces a _Review_ feature, that empowers Flutter teams to c
 
 ## Overview
 
-1. [Understanding Reviews](/widgetbook-cloud/reviews#understanding-reviews)
-2. [Review Statuses](/widgetbook-cloud/reviews#review-statuses)
-3. [Requirements](/widgetbook-cloud/reviews#requirements)
+1. [Concept of reviews](/widgetbook-cloud/reviews#understanding-reviews)
+2. [Publish a review](/widgetbook-cloud/reviews#publishing-a-review)
+3. [Review Statuses](/widgetbook-cloud/reviews#review-statuses)
 4. [Set up addons and knobs](/widgetbook-cloud/reviews#how-to-set-up-addons-and-knobs)
 5. [Add a Figma design link](/widgetbook-cloud/reviews#how-to-add-a-figma-design-link)
-6. [Publishing a review](/widgetbook-cloud/reviews#publishing-a-review)
+
 
 ---
 
-## Understanding Reviews
+## Concept of reviews
 
 A Widgetbook Review lets teams contrast UI changes between two versions. Moreover, it enables comparison of the current Widget version against its Figma design.
 - In a standard Pull Request, code differences between a `base` (previous version) and a `feature` (new version) branch are evaluated.
 - Similarly, in a Widgetbook Review, visual differences between the base build (last version) and feature build (current version) are assessed.
 For every pull request in your Git provider, our integration initiates a Widgetbook Cloud review. To facilitate this visual comparison, Widgetbook Cloud necessitates uploaded builds for both base and feature branches.
 
+## Publish a Review
+
+1. Make sure to incorporate the [widgetbook_generator package](https://pub.dev/packages/widgetbook_generator) version `{{ versions.generator }}`. No additional configuration needed.
+2. Integrate the [widgetbook package](https://pub.dev/packages/widgetbook) version `{{ versions.widgetbook }}`. No extra setup required.
+3. Employ the CLI from [widgetbook_cli package](https://pub.dev/packages/widgetbook_cli).
+4. Set up our GitHub app.
+5. Upload a build of your base branch, e.g. main. See [how to upload a build](https://docs.widgetbook.io/widgetbook-cloud/hosting).
```

Comment:
main should be formatted as `main`. 

The link must be relative and must not contain `https:`